readthedocs/core/views/__init__.py

```diff
@@ -23,6 +23,11 @@ class NoProjectException(Exception):
     pass
 
 
+class HealthCheckView(View):
+    def get(self, request, *args, **kwargs):
+        return JsonResponse({'ok'}, status=200)
```

Comment:
Probably a dict here instead? I'm assuming it's getting encoded to an array in JSON output

```suggestion
        return JsonResponse({'status': 'ok'}, status=200)
```